hack/conformance.sh

```diff
@@ -1,8 +1,10 @@
-set -xe
+#!/bin/bash
+
+set -xeuo pipefail
 
 ARTIFACTS=${ARTIFACTS:-${PWD}}
 sonobuoy_version=0.18.2
-config_file=$1
+config_file=${1:-}
```

Comment:
Shouldnt we need to check if the file exists, here or at caller function line 44, and only then continue?

Reply:
it allows to override the conformance_config env var, 
and the change i did here is because i added set -u